libparsec/crates/platform_mountpoint/src/unix/filesystem.rs

```diff
@@ -1121,7 +1113,23 @@ impl fuser::Filesystem for Filesystem {
 
         // TODO: support atime/utime change ?
 
-        reply.manual().error(libc::ENOSYS);
+        // Nothing to set, just return the file attr
```

Comment:
The comment is a bit off since the previous one stated that there is things to change BUT we don't support them

Reply:
For me it's nothing to set on parsec side either because we ignore them or we don't handle them.
I'm open to suggestion to improve the comment

Reply:
I'd also mention #8976 and #8991 here in the comment because it is not evident that this was needed to fix those problems.